scripts/package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# package.sh by Andrzej Pogonowski
+# What is this?: This is the script that packages all the necessary files for deployment
+# Why does this exists?: Apparently jenkins agents dump core and memory information into the directory
+# each of these files is 40MB, adding node_modules and the build folder the resulting zip file is 40MB
+# This bash script solves the problem by creating a folder and copying only the required folders 
+
+# Steps
+# 1. Removes all of the build and dependency directories (build, pack, node_modules)
+# 2. Copies all of the source files
+# 3. Copes configuration files and other necessary files
+
+# Cleanup
+rm -r pack
+rm -r build
+rm -r node_modules
+mkdir pack
+chmod -R 755 pack
+
+# Copy needed directories
```

Comment:
Yes, I saw this issue as well. This script will most likely be only temporary since I want to build a more intelligent one anyway. The script will copy the entire `/src` folder so 99% of changes will be captured by this script. It is only a problem when you are adding the files to the root directory. In the meantime I will add the relevant docs